plugin/adsk/scripts/mayaUsd_createStagesFromExistingLayer.mel

```diff
@@ -71,19 +86,99 @@ proc doCreateStage(string $fileName) {
     else
         $baseName = "UsdStage";
 
-    global string $stageFromExistingLayer_primPath;
-    global string $stageFromExistingLayer_excludePrimPath;
-    global int $stageFromExistingLayer_loadPayloads;
+    string $ppath = `optionVar -q stageFromExistingLayer_primPath`;
+    string $exppath = `optionVar -q stageFromExistingLayer_excludePrimPath`;
+    int $loadp = `optionVar -q stageFromExistingLayer_loadPayloads`;
 
     string $shapeNode = `createNode "mayaUsdProxyShape" -skipSelect -name ($baseName+"Shape")`;
     setAttr -type "string" ($shapeNode+".filePath") $fileName;
-    setAttr -type "string" ($shapeNode+".primPath") $stageFromExistingLayer_primPath;
-    setAttr -type "string" ($shapeNode+".excludePrimPaths") $stageFromExistingLayer_excludePrimPath;
-    setAttr ($shapeNode+".loadPayloads") $stageFromExistingLayer_loadPayloads;
+    setAttr -type "string" ($shapeNode+".primPath") $ppath;
+    setAttr -type "string" ($shapeNode+".excludePrimPaths") $exppath;
+    setAttr ($shapeNode+".loadPayloads") $loadp;
     connectAttr time1.outTime ($shapeNode+".time");
     select -r $shapeNode;
 }
 
+global proc createUSdStagesFromExistingCallback(string $parent, int $doIt)
```

Comment:
The `USd` in the function name looks a little odd here and in the function below, and seems a little inconsistent with other functions in this file that just say `createStage...` or whatever. Maybe just drop the "USd"?

Reply:
This is what happens when you type something by hand and then copy/paste it around.  Also like removing the "USD" entirely to match other functions.